src/test/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2IntegrationTest.java

```diff
@@ -660,6 +661,11 @@ public void testFilterMitochondria() throws Exception {
         }
     }
 
+    @Test
+    public void testFilterMitochondriaGVCFs() throws Exception {
+
```

Comment:
This test is empty.

Reply:
Ugh, I am the worst at stashing commits.  That should be on a different branch.

Reply:
In that case, :thumbsup:.

Reply:
Now three changed files!  I will merge when tests pass.